index.html

```diff
@@ -8,152 +8,219 @@
+                    <h2 class="desktop">Uber Navigation</h2>
+                    <div class="frame mobile">
+                        <ul>
+                            <li class="canopy">CANOPY</li>
+                            <li class="separator"></li>
+                            <li>Back End Dev</li>
+                            <li class="separator"></li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <div class="frame desktop">
+                        <ul>
+                            <li class="canopy">Uber</li>
+                            <li class="separator"></li>
+                            <li>Lead Developer</li>
+                            <li class="separator"></li>
+                            <li>2018</li>
+                        </ul>
+                    </div>
+                    <p class="mobile">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <p class="desktop">A smart assistant to make driving more safe, efficient, and fun by unlocking your
+                        most expensive computer: your car.</p>
+                    <div class="tags">
+                        <ul>
+                            <li><button type="button">html</button></li>
+                            <li class="tag desktop"><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">css</button></li>
+                            <li><button type="button">javascript</button></li>
+                        </ul>
+                    </div>
+                    <div class="see_project_button">
+                        <button type="button">See Project</button>
+                    </div>
                 </div>
             </div>
         </section>
         <section class="About_Myself_ Section">
             <div class="content_2">
-                <h2>About <br> Myself</h2>
-                <p>Hello I’m a software developer! I can help you build a product, feature or website Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don’t
-                    hestiate to contact me.</p>
-                <h6><a href="#">LET’S CONNECT</a></h6>
-                <div class="social-media-icons">
-                    <ul class="social-media-icons">
+                <div class="split">
+                    <h2>About <br> Myself</h2>
+                    <p>Hello I’m a software developer! I can help you build a product, feature or website Look through
+                        some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me.</p>
+                    <h6><a href="#">LET’S CONNECT</a></h6>
+                    <ul class="social-media-icons_2">
```

Comment:
- [ ] I would suggest you increase the `padding-top` for your `about` section and also remove this empty blank bottom space to stick the Figma design.
**1. Your design**
![Screenshot from 2022-07-09 14-24-25](https://user-images.githubusercontent.com/80612925/178105931-38c82441-3b80-4246-a3c2-ef298422c424.png)

**2. Figma design:**
![Screenshot from 2022-07-09 14-33-41](https://user-images.githubusercontent.com/80612925/178105994-e31e3d78-258d-4e2c-b071-a39fa0673fbb.png)